tools/bin/gpcheck

```diff
@@ -567,7 +683,7 @@ def testHAWQ(host):
             return
 
         # check HAWQ master's memory size
-        expected_vmemory_size = 1024
+        expected_vmemory_size = 8192
```

Comment:
If user set the value too big, like set it as '81920' by mistake, 'hawq check' won't give a warning since it's bigger than the expect values.

So I think we can keep current way. User will decide wether to fix a hawq check warning base on the document and their environment. 

If any new requirement, we can change it in another jira. Thanks.